src/panels/config/zwave/zwave-node-config.js

```diff
@@ -178,12 +192,54 @@ class ZwaveNodeConfig extends PolymerElement {
         type: Boolean,
         value: false,
       },
+
+      protectionNode: {
```

Comment:
All these protection options and html should be extracted in their own web component